docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_GluonCV_Models.md

```diff
@@ -0,0 +1,25 @@
+You can convert [SSD and YOLO-v3 models from GluonCV](https://gluon-cv.mxnet.io/model_zoo/detection.html) to IR using the following instructions:
+
+1 Chouse topology and export it to MXNet format:
+```python
+from gluoncv import model_zoo, data, utils
+from gluoncv.utils import export_block
+net = model_zoo.get_model('ssd_512_mobilenet1.0_voc', pretrained=True)
+export_block('ssd_512_mobilenet1.0_voc', net, preprocess=True, layout='HWC')
+```
+
+2 Convert model using Model Optimizer tool:
+* For GluonCV SSD topologies:
+```sh
+python3 mo_mxnet.py --input_model ~/models/ssd_300_t_mobile-0000.params --enable_ssd_gluoncv --input_shape [1,3,480,640] --input data
```

Comment:
Are you sure this works? In my experience, I have had to use another ordering: `--input_shape [1,480,640,3]`